docs/moment/01-parsing/03-string-format.md

```diff
@@ -23,6 +23,13 @@ moment("12-25-1995", "MM-DD-YYYY");
 moment("12/25/1995", "MM-DD-YYYY");
 ```
 
+The parser considers spaces to not be non-alphanumeric, so neither of the following will parse as expected:
+
+```javascript
+moment("12 25 1995", "MM-DD-YYYY");
+moment("12 25 1995", "MM/DD/YYYY");
```

Comment:
Are you sure? I ran this code and it evaluated fine for me.

Reply:
It's actually more complicated, and I forgot by how much more when I commit this code. I'll update with the edge case as soon as I can.

In the meantime check the issue linked.